libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -6,10 +6,13 @@ import { DesignTokenHelper } from '@kirbydesign/designsystem/helpers';
 
 import { IconModule } from '@kirbydesign/designsystem/icon';
 
+import { TestHelper } from 'testing/src';
```

Comment:
Probably doesn't matter but everywhere else we use the absolute import:
```suggestion
import { TestHelper } from '@kirbydesign/designsystem/testing';
```